### PR TITLE
navbar with working routing

### PR DESCRIPTION
**Pull request by himanshu1221**
Added a navbar in dark theme with complete working routing menthod and they are switching pages without reload there are some error with files practice.js & analyisis.js they are not loading but routing tags and contest are woriking fluently.
.
would be glag if u merge it and give the label of **KWoc22**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -25,6 +25,7 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65" crossorigin="anonymous">
```
Comment (shrinish123) [inline public/index.html:28]: Why is bootstrapped imported if you are using material ui

Comment (himanshu1221) [reply public/index.html:28]: I told u before getting assigned i used bootstrap to make navbar ig i forgot to delete the scripts``

**Comment by shrinish123** [comment]
Right now the way we are managing state and data, we probably would not be able to load pages like contests, analysis, and suggested practice. So just don't add these pages for now. 
Also the contributions are going to counted on the develop branch, so make a new PR to develop branch. right now its to the main branch . 
Also the coding period starts from 7th December, when the contributions start counting. 
So make the suggested changes and then will merge them after proper completion of the PR.

**Comment by himanshu1221** [comment]
> Right now the way we are managing state and data, we probably would not be able to load pages like contests, analysis, and suggested practice. So just don't add these pages for now. Also the contributions are going to counted on the develop branch, so make a new PR to develop branch. right now its to the main branch . Also the coding period starts from 7th December, when the contributions start counting. So make the suggested changes and then will merge them after proper completion of the PR.

sure !

**Pull request closed without merging**